opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -361,36 +361,30 @@ def _extract_resources(resource: Resource) -> Dict[str, str]:
     }
 
 
-PREDIFINED_LABELS = [
-    "/agent",
-    "/component",
-    "/error/message",
-    "/error/name",
-    "/http/client_city",
-    "/http/client_country",
-    "/http/client_protocol",
-    "/http/client_region",
-    "/http/host",
-    "/http/method",
-    "/http/path",
-    "/http/redirected_url",
-    "/http/request/size",
-    "/http/response/size",
-    "/http/route",
-    "/http/status_code",
-    "/http/url",
-    "/http/user_agent",
-    "/pid",
-    "/stacktrace",
-    "/tid",
-]
-
-
-def _key_predefined(key: str):
-    result = "/%s" % key.replace(".", "/")
-    if result in PREDIFINED_LABELS:
-        return result
-    return key
+LABELS_MAPPING = {
+    # "agent": "/agent",
+    "component": "/component",
+    "error.message": "/error/message",
+    "error.name": "/error/name",
```

Comment:
one could argue the following could be used as well, not sure in fact as the google documentation is scarce about the meaning they give to the labels.

`exception.message` for `/error/message`
`exception.type` for `/error/name`
`exception.stacktrace` for `/stacktrace`

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/exceptions.md#attributes

Reply:
I think `span.status` would be most appropriate for mapping to `/error/message` and `/error/name` because it is a single field meant to classify the *entire* span, while you can have multiple exception `Event`s and they don't necessarily mean an error.

I'm not sure how that fits with `/stacktrace`, since you can have multiple exceptions. This could use some clarification from someone on Cloud Trace.

Reply:
ok, I removed them